nupic/frameworks/opf/clamodel.py

```diff
@@ -441,6 +443,21 @@ def _getSensorInputRecord(self, inputRecord):
                        sequenceReset=resetOut,
                        category=inputRecordCategory)
 
+  def _getClassifierInputRecord(self, inputRecord):
```

Comment:
Is this new method required for implementing this metric, or is this for something else?